server/routes/rpaFramework.js

```diff
@@ -17,6 +17,11 @@ router.get(
   commandsController.getAvailableTasksForApplications
 );
 
+router.get(
+  '/commands/getAllParameters',
+  commandsController.getAllParameters
+);
+
 router.get('/commands/get-vars-for-task', commandsController.getVarsForTask);
```

Comment:
The routes should be consistent in their naming. Either camelCase or hyphens.

Reply:
Routes will be renamed in a future issue.